docs/core/dependency-loading/collect-details.md

```diff
@@ -24,6 +26,24 @@ dotnet-trace collect --providers Microsoft-Windows-DotNETRuntime:4 --process-id
 
 This will collect a trace of the specified `<pid>`, enabling the `AssemblyLoader` events in the `Microsoft-Windows-DotNETRuntime` provider. The result is a `.nettrace` file.
 
+### Use dotnet-trace to launch a child process and trace it from startup
```

Comment:
While I appreciate the desire to have only one doc to follow, I do think this should include a link to the documentation for `dotnet-trace` which has much more detail in it (and is likely to be more up to date, inevitably this doc will get stale as it's a copy of the stuff from the `dotnet-trace` doc).

Reply:
@vitek-karas Why would it get stale?

I can't tell from your feedback what you want done exactly.  This page addresses the specific use case of how to collect details around dependency loading.  This is a pretty big user story.  The fact that `dotnet-trace` is its own tool that collapses assembly fusion loading story doesn't change the fact there is a need for a page to clearly explain the big user story.  I think rather than debate one vs. two pages the debate should focus on search engine optimization to make sure common search terms for this problem actually direct people to this page.  When you dedicate a page to a user story, nobody can then come along and just wipe out the content without realizing the context.  Just my 2 cents.

Reply:
As I said I think it's a good idea to keep a simple guide how to do things on this one page. But I do believe it should also contain something like "For more details on usage of `dotnet-trace` please refer to https://docs.microsoft.com/en-us/dotnet/core/diagnostics/dotnet-trace" (or if there's a section more appropriate then point to that).

As for it getting stale. People who work on `dotnet-trace` will update its documentation, but they are not going to remember to update this page if needed. It's not a huge concern as I don't think we would change the command line options to `dotnet-trace` without a really good reason, but still - it's at least worth the additional link.

Reply:
Got it.  I will update to include the link back to dotnet-trace.  I think I misunderstood what you were saying, forgive me.

Reply:
I added a note under the prerequisites section.